File: esn_tutorial/src/limit_switch_node.cpp
#include <esn_tutorial/limit_switch.hpp>

int main(int argc, char ** argv)
{
    rclcpp::init(argc, argv);
    auto node = std::make_shared<esn::LimitSwitch>();
    rclcpp::spin(node);
    rclcpp::shutdown();
    return 0;
}

File: esn_tutorial_py/esn_tutorial_py/limit_switch_node.py
import rclpy
from esn_tutorial_py.limit_switch import LimitSwitch


def main(args=None):
    rclpy.init(args=args)
    node = LimitSwitch()

    try:
        rclpy.spin(node)
    except KeyboardInterrupt:
        pass
    finally:
        node.destroy_node()
        rclpy.shutdown()


if __name__ == '__main__':
    main()
